openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -401,7 +401,7 @@ def test_patch_account(self, field, value, fails_validation_value=None, develope
             self.send_patch(client, {field: ""})
 
             get_response = self.send_get(client)
-            self.assertEqual("", get_response.data[field])
+            self.assertFalse("", get_response.data[field])
```

Comment:
This was a poor hack to fix a test (and one that I screwed up to boot). Christina gave me a better solution.